src/default-reducers.js

```diff
@@ -94,53 +127,53 @@ export function retrieveManyAborted(state, action) {
   };
 }
 
-export function retrieveManyResetResolution(state, action) {
+export function retrieveManyResetResolution(idAttr, state, action) {
   return {
     ...state,
     resourcesListMeta: {
-      retrievingStatus: null
+      retrievingStatus: resourceStatuses.NULL
     }
   };
 }
 
 // Update reducers
-export function update(state, action) {
+export function update(idAttr, state, action) {
   console.log('update');
 }
 
-export function updateFailure(state, action) {
+export function updateFailure(idAttr, state, action) {
   console.log('update_failure');
 }
 
-export function updateSuccess(state, action) {
+export function updateSuccess(idAttr, state, action) {
   console.log('update_success');
 }
 
-export function updateAborted(state, action) {
+export function updateAborted(idAttr, state, action) {
   console.log('update_aborted');
 }
 
-export function updateResetResolution(state, action) {
+export function updateResetResolution(idAttr, state, action) {
   console.log('update_reset_resolution');
 }
 
 // Delete reducers
-export function del(state, action) {
+export function del(idAttr, state, action) {
   console.log('del');
 }
 
-export function delFailure(state, action) {
+export function delFailure(idAttr, state, action) {
   console.log('del_failure');
 }
 
-export function delSuccess(state, action) {
+export function delSuccess(idAttr, state, action) {
   console.log('del_success');
 }
 
-export function delAborted(state, action) {
+export function delAborted(idAttr, state, action) {
   console.log('del_aborted');
 }
 
-export function delResetResolution(state, action) {
+export function delResetResolution(idAttr, state, action) {
```

Comment:
Resources are usually referenced as having IDs, and typically not keys, so I want "id" in there. I went with `idAttr` for brevity since it's used so much, and sometimes in long lines, but I can try out `idAttribute` and see how it goes if you'd like.